history-of-quakers/modernized/21-chapter-19.adoc

```diff
@@ -1088,7 +1120,8 @@ which goes to the completing of the Christian religion;
 as the hands or feet of a man are integral parts of a man,
 without which nevertheless a man may exist, but not an entire and complete man.
 
-Secondly, If by immediate revelation be understood such a revelation of God,
+[.numbered]
```

Comment:
I wasn't sure if I should [.number] this, but it didn't really look right without it in the PDF though.

Reply:
Seems good to me this way as well. 👍